spot-ml/ml_ops.sh

```diff
@@ -118,7 +120,8 @@ time spark-submit --class "org.apache.spot.SuspiciousConnects" \
   --nodes ${nodes} \
   --scored ${HDFS_SCORED_CONNECTS} \
   --threshold ${TOL} \
-  --maxresults ${MAXRESULTS}
+  --maxresults ${MAXRESULTS}\
+  --ldaImplementation ${LDA_IMP}
```

Comment:
even if lda-c stays, maybe we could have some root name from which all the other lda-c temp files (model, topicdoc, topicword) get their names...

but hopefully it doesn't stay